### PR TITLE
Update Sarama baseline version to 1.27.2

### PR DESCRIPTION
**Pull request by faec**
This is a simple merge with the current Sarama release. It was prompted by authentication problems in Beats that are partially addressed by the new Kerberos configuration options available in version 1.27.0+.

### CODE REVIEW

**Comment by andresrc** [comment]
ping @elastic/agent

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -9,18 +9,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.14.x]
-        kafka-version: [2.4.1, 2.5.0]
+        go-version: [1.15.x]
```
Comment (urso) [inline .github/workflows/ci.yml:12]: Why update go version?

**Comment by urso** [comment]
Do we require CI to run here?

**Comment by faec** [comment]
No, I'm surprised to even see it trying to run CI, it seems to be trying to run Shopify's scripts? I don't remember seeing that before, or the CLA error, I wonder if something about the Github repo configuration changed.

**Review by kvch — APPROVED** [review]

**Pull request merged**